pkg/colorful-portuguese.man

```diff
@@ -0,0 +1,31 @@
+.\" Manpage para colorful
+.\" Entre em contato com veg@svgames.pl para corrigir erros de digitação.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```

Comment:
"Games Manual" should also be translated. The date should also reflect the file modification date.

Reply:
Ok... It has been fixed.